src/wp-admin/includes/class-wp-community-events.php

```diff
@@ -468,6 +469,9 @@ protected function trim_events( array $events ) {
 			$end_time = (int) $event['end_unix_timestamp'];
 
 			if ( time() < $end_time ) {
+				// Decode HTML entities from the event title.
+				$event['title'] = html_entity_decode( $event['title'] );
```

Comment:
```suggestion
				$event['title'] = html_entity_decode( $event['title'], ENT_QUOTES, 'UTF-8' );
```

Shouldn't quotes be decoded too? For consistency with similar calls we should also explicitly set the charset.

Reply:
@ocean90 ah yes good point, thanks.
Question: shouldn't we use `get_option( 'blog_charset' )` instead of `'UTF-8'`.

Reply:
Core isn't super strict in that terms, but since `blog_charset` is `UTF-8` by default either is okay I guess.